mobile/apps/artwork/client/index.coffee

```diff
@@ -15,7 +15,7 @@ module.exports.init = ->
   artworkTabsView()
   artwork = new Artwork sd.ARTWORK
   user = CurrentUser.orNull()
-  user.initializeDefaultArtworkCollection()
+  user.initializeDefaultArtworkCollection() if user
```

Comment:
This was added in https://github.com/artsy/force/commit/6b41215342d320d83aa58d3d62813e76aea9a297 (merger).